nncf/torch/hardware/fused_patterns.py

```diff
@@ -53,16 +58,24 @@ def create_l2_norm_operations() -> GraphPattern:
 @PT_HW_FUSED_PATTERNS.register(HWFusedPatternNames.SHIFT_SCALE)
 def create_shift_scale() -> GraphPattern:
     pattern = GraphPattern()
-    add_node = pattern.add_node(label="ADD, SUB", type=["__add__", "__sub__"])
-    truediv_node = pattern.add_node(label="MUL, DIV", type=["__mul__", "__truediv__"])
+    add_node = pattern.add_node(label="ADD, SUB", type=[PTAddMetatype, PTSubMetatype])
```

Comment:
@ljaljushkin please review changes in shift_scale patters

Found that pattern does not match to test case https://github.com/openvinotoolkit/nncf/blob/242aacdcb82035d7035476812099f8d8bf3cce03/tests/torch/test_models/synthetic.py#L367
1. `div_` and `sub_` was not matched to any metatype as result FQ was not added (not by patter)
2. Pattern does not match because of the `clone` node exists in the graph
3. Pattern does not match because of the 'is_floating_point' branch exists in the graph

Reply:
Agreed that FQ shouldn't be inserted between in-place add and mul operations. 
Some models may use this pattern https://github.com/PeterL1n/RobustVideoMatting/blob/master/model/mobilenetv3.py#L37

Reply:
Reverted changes for inplace ops.